### PR TITLE
Scorching Ray writeup, leading spaces in desc fix

### PR DESCRIPTION
**Pull request by antihumorous**
## What does this do?
This update changes some leading spaces in spell descriptions for consistency. Antimagic Field's description was updated for consistency with how other spells handled bolded description headers. It also updates the description for Scorching Ray, which was transcribed incorrectly.

## How was it tested?
I ran a regex search and replace and did a quick look over. I also tested it by rendering the descriptions with my chat bot.

## Is there a Github issue this is resolving?
Not that I know of.

## Here's a fun image for your troubles
random photo - update me

### CODE REVIEW

File: src/5e-SRD-Spells.json [hunk 1]
```diff
@@ -9889,11 +9896,11 @@
     "name": "Scrying",
     "desc": [
       "You can see and hear a particular creature you choose that is on the same plane of existence as you. The target must make a wisdom saving throw, which is modified by how well you know the target and the sort of physical connection you have to it. If a target knows you're casting this spell, it can fail the saving throw voluntarily if it wants to be observed.",
-      "Knowledge &amp; Save Modifier\n",
+      "Knowledge &amp; Save Modifier",
```
Comment (bagelbits) [inline src/5e-SRD-Spells.json]: Not on you but should this be `&amp;`?

File: src/5e-SRD-Spells.json [hunk 2]
```diff
@@ -9889,11 +9896,11 @@
...
       "Secondhand (you have heard of the target) +5",
       "Firsthand (you have met the target) +0",
       "Familiar (you know the target well) -5",
-      "Connection &amp; Save Modifier\n",
+      "Connection &amp; Save Modifier",
```
Comment (bagelbits) [inline src/5e-SRD-Spells.json]: Same here?

**Comment by bagelbits** [comment]
@antihumorous Small comment but otherwise looks great!

File: src/5e-SRD-Spells.json (hunk 2 above)
Comment (antihumorous) [reply src/5e-SRD-Spells.json]: Probably a Javascript parsing thing. I believe it's all the same, but we can change them to strict ampersands if you prefer. Either way, I'm good if you wanna merge this. :)

Comment (bagelbits) [reply src/5e-SRD-Spells.json]: Yeah, do you mind converting them to strict ampersands? Then it's more consistent. I'll merge it in after you do that.

**Comment by antihumorous** [comment]
All done!

**Review by bagelbits — APPROVED** [review]
LGTM!

**Pull request merged**